### DISCUSSION TITLE
Comprei um template maneiro, e agora como componentizar sem ter trabalho?

### DISCUSSION THREAD

**Discussion by inNetto** [frontendbr/forum#217] [Perguntas e Respostas]
Opa rapaziada. Comecei a seguir o grupo agora e já vim com essa dúvida na qual não sei a resposta. 
Minha experiência dessa semana foi comprar um template (metronic, pra quem conhece .. show de bola) e tomar uma surra pra poder criar meus componentes e estilizar conforme o template. Qual a melhor forma de se componentizar templates prontos? Como vocês fazem? 
Agradeço ao grupo. abraço

**Comment by inNetto** [comment]
faltou falar o principal rs ... componentizar com VUE JS

**Comment by bernardodiasc** [comment]
Tem muitas formas diferentes de arquitetar um frontend, depende muito do template que voce comprou e de como o autor o fez. Talvez voce tenha que desmanchar o template inteiro em partes menores e depois adaptar pro framework que voce quer.

Havia visto uma pagina mostrando a comparação entre os componentes em diferentes frameworks js, mas nao consegui achar o link.

Recomendo a leitura deste http://elijahmanor.com/talks/plain-js-comps/, talvez seja uma boa abordagem para o que voce precisa.
